src/haddock/modules/scoring/sasascore/__init__.py

```diff
@@ -0,0 +1,143 @@
+"""
+Surface accessibility scoring module.
+
+This module performs a solvent accessibility analysis based on 
+some user-defined residues that should be buried or accessible.
+
+If a supposedly buried (resp. accessible) residue is accessible (resp. buried),
+the score should increase by one. The lower the final score the more consistent
+the model with the user data.
+"""
+from pathlib import Path
+
+from haddock.core.typing import FilePath
+from haddock.modules import get_engine
+from haddock.modules import BaseHaddockModule
+from haddock.libs.libutil import parse_ncores
+from haddock.libs.libparallel import get_index_list
+from haddock.modules.scoring.sasascore.sasascore import (
+    AccScore,
+    AccScoreJob,
+    prettify_df,
+    )
+from haddock.modules.analysis import (
+    get_analysis_exec_mode,
+    )
+
+RECIPE_PATH = Path(__file__).resolve().parent
+DEFAULT_CONFIG = Path(RECIPE_PATH, "defaults.yaml")
+
+
+class HaddockModule(BaseHaddockModule):
+    """HADDOCK3 module to perform accessibility scoring."""
+
+    name = RECIPE_PATH.name
+
+    def __init__(self,
+                 order: int,
+                 path: Path,
+                 initial_params: FilePath = DEFAULT_CONFIG) -> None:
+        super().__init__(order, path, initial_params)
+
+    @classmethod
+    def confirm_installation(cls) -> None:
+        """Confirm module is installed."""
+        return
+
+    def _rearrange_output(self, output_name: FilePath, path: FilePath,
```

Comment:
I would put the reading/rearranging for the tsv files in the `sasascore.py` script.
Maybe just keep this method here for the `self.log()` wrapping the thing ?

Reply:
Exactly, this method's only self attribute is the logging, best to move it to the `sasascore.py` module and log it from there